client/src/components/Services.tsx

```diff
@@ -4,23 +4,24 @@ import { BASE_URL } from '../constants';
 
 
 function GetData(): string[] {
-	const [data, setData] = useState<any[]>([]);
-	const serviceNames: string[] = [];
+    const [data, setData] = useState<any[]>([]);
+    const serviceIds: string[] = [];
```

Comment:
I think it's better to just store the entire array of service objects, so we can display the name as well. It looks like there's a `Service` type that might be out of date, but it should still work.

But, I also feel like this Services page is redundant since we're showing the exact same services as on the home page, except the home page uses service cards. BTW, those service cards are missing the link to the individual full pages, so we could just move the links over... I'll leave it up to you on what changes you want to make here; we could always address the other stuff in a different ticket.

Reply:
Yeah for sure, I was under the impression that you wanted the refactoring to be part of a different ticket that's why I didn't add it here. I'll update that. As for the service cards, I can try adding the links and I'm guessing they'll be displayed on the home page. So then do we want to get rid of the services tab all together?

Reply:
Ahh right, yeah that refactoring ticket could cover this too. Although, if we're getting rid of the services page, maybe there isn't much to do here..

I think it makes sense to ignore/hide the services page for now, since it doesn't serve any other meaningful purpose. There's a few other placeholder views and components (sign-in button, about page, reviews, etc.) that we should also hide, so we could do all this in another ticket instead. So maybe all there is to do is just to add the service card links?